packages/g/gstreamer-0.10/pspec_x86_64.xml

```diff
@@ -56,44 +57,44 @@
-            <Path fileType="localedata">/usr/share/locale/en_GB/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/eo/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/es/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/eu/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/fi/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/fr/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/gl/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/hu/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/id/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/it/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/ja/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/lt/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/nb/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/nl/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/pl/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/pt_BR/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/ro/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/ru/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/rw/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/sk/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/sl/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/sq/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/sr/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/sv/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/tr/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/uk/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/vi/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/zh_CN/LC_MESSAGES/gstreamer-0.10.mo</Path>
-            <Path fileType="localedata">/usr/share/locale/zh_TW/LC_MESSAGES/gstreamer-0.10.mo</Path>
+            <Path fileType="localedata">/usr/share/locale/af/LC_MESSAGES/.mo</Path>
```

Comment:
These locale filepaths look wrong, this would be a single file called `.mo` 
Can you see what's going on here?

Reply:
I have been looking around but can't seem to find any resolution here.